nb-extension/js/init/init.js

```diff
@@ -138,16 +138,20 @@ define([
         });
     }
 
+    ////////////////////////////////////////////////////////////////////////////
+
+    var DeclWidgets = {};
+
     /**
      * Initialize Declarative Widgets
      * @param  {Object} config - configuration; see following entries for more information
      * @param  {Object} config.namespace - Jupyter Notebook namespace object (or shim)
      * @param  {Object} config.events - Notebook events object
-     * @param  {Object} config.WidgetManager - widget manager class
+     * @param  {Object} [config.WidgetManager] - widget manager class (for backwards compatibility)
```

Comment:
I kept this for "backwards compatibility", but I think that only affects the previous commit I did (removing `IPython` deps). Did that code ever make it to release? If not, then no reason to keep this around.